cmd/govim/watcher.go

```diff
@@ -165,6 +87,39 @@ func (m *modWatcher) watch() {
 	}
 }
 
+func eventFilter(root string) func(string) bool {
+	return func(path string) bool {
+		if fi, err := os.Stat(path); err == nil && !fi.IsDir() {
+			path = filepath.Dir(path)
+		}
+
+		// TODO: cache ignored directories to reduce amount of os.Stat(...) calls
+		rel := strings.TrimPrefix(path, root)
+		parts := strings.Split(rel, string(os.PathSeparator))
+		if len(parts) == 0 {
+			return false
+		}
+
+		curr := parts[0]
+		for i := 1; i < len(parts); i++ {
+			dir := parts[i]
+			switch dir[0] {
+			case '.', '_':
```

Comment:
Can we link to the relevant part of the Go source code/docs for the reference on this list of conditions?

Reply:
Good idea, will do.

Reply:
As note to self - https://golang.org/cmd/go/#hdr-Package_lists_and_patterns